modules/qtwidgets/src/rangesliderqt.cpp

```diff
@@ -295,17 +296,17 @@ bool RangeSliderQt::eventFilter(QObject* obj, QEvent* event) {
         }
     } else if (isEnabled()) {
         if (obj == widget(0)) {
-            handleGrooveEvent(Groove::Left, event);
+            return handleGrooveEvent(Groove::Left, event);
         } else if (obj == widget(1)) {
-            handleCenterWidgetEvent(event);
+            return handleCenterWidgetEvent(event);
         } else if (obj == widget(2)) {
-            handleGrooveEvent(Groove::Right, event);
+            return handleGrooveEvent(Groove::Right, event);
```

Comment:
Could check if event is handled afterwards instead? Returning the bool has the same meaning?

Reply:
I don't now, for an event filter then return value is important. And this fixed the problem I had.